core-api/src/main/java/com/optimizely/ab/config/Variation.java

```diff
@@ -37,19 +37,28 @@
 
     private final String id;
     private final String key;
+    private final Boolean featureEnabled;
     private final List<LiveVariableUsageInstance> liveVariableUsageInstances;
     private final Map<String, LiveVariableUsageInstance> variableIdToLiveVariableUsageInstanceMap;
 
     public Variation(String id, String key) {
         this(id, key, null);
     }
 
+    public Variation(String id,
+                     String key,
+                     List<LiveVariableUsageInstance> liveVariableUsageInstances) {
+        this(id, key,false, liveVariableUsageInstances);
+    }
+
     @JsonCreator
     public Variation(@JsonProperty("id") String id,
                      @JsonProperty("key") String key,
+                     @JsonProperty("featureEnabled") Boolean featureEnabled,
                      @JsonProperty("variables") List<LiveVariableUsageInstance> liveVariableUsageInstances) {
         this.id = id;
         this.key = key;
+        this.featureEnabled = featureEnabled;
```

Comment:
I see in the JSON parsers you make sure that the featureEnabled is not null.  Can't you do that here as well?  if (featureEnbaled == null)  set it to false.  Just for safety sake.